src/renderer/components/App/AppIntro/AppIntro.vue

```diff
@@ -0,0 +1,296 @@
+        v-if="step === 0"
+        :show-back="false"
+        :show-next="false"
+        :show-skip="false"
+        :show-logo="false"
+        :image="stepImage"
+        class="AppIntro__1"
+      >
+        <div
+          slot="content"
+          class="w-2/3 font-medium"
+        >
+          <i18n
+            path="INTRODUCTION.WELCOME.TITLE"
+            tag="div"
+            class="font-bold mb-4"
+          >
+            <p
+              place="APP"
+              class="font-black text-3xl"
+            >
+              {{ $t('COMMON.APP_NAME_SHORT') }}
+            </p>
+          </i18n>
+          <p>
+            {{ $t('INTRODUCTION.WELCOME.SAFETY_MESSAGE') }}
+          </p>
+          <p class="mt-2">
+            {{ $t('INTRODUCTION.WELCOME.FUNDS_WARNING') }}
+          </p>
+
+          <div
+            class="flex flex-row align-center justify-center mt-8"
+          >
+            <MenuDropdown
+              :items="languages"
+              :value="language"
+              :position="['-50%', '0%']"
+              class="AppIntro__1__languages flex align-center justify-center p-2 text-grey-dark"
+              @select="selectLanguage"
+            >
+              <div
+                slot="item"
+                slot-scope="itemScope"
+                class="flex flex-row space-between"
+              >
+                <img
+                  :src="flagImage(itemScope.value)"
+                  :title="itemScope.item"
+                  class="AppIntro__1__languages__item__flag mr-2"
+                >
+                <span class="font-semibold">
+                  {{ itemScope.item }}
+                </span>
+              </div>
+
+              <div
+                slot="handler"
+                slot-scope="handlerScope"
+              >
+                <MenuDropdownHandler
+                  :value="handlerScope.activeValue"
+                  :item="handlerScope.item"
+                  :placeholder="handlerScope.placeholder"
+                  :prefix="handlerScope.prefix"
+                  :icon-disabled="handlerScope.isOnlySelectedItem"
+                >
+                  <img
+                    :src="flagImage(handlerScope.value)"
+                    :title="handlerScope.item"
+                    class="AppIntro__1__languages__handler__flag mr-1"
+                  >
+                  {{ handlerScope.item }}
+                </MenuDropdownHandler>
+              </div>
+            </MenuDropdown>
+
+            <ButtonGeneric
+              :label="$t('COMMON.START')"
+              class="ml-4"
+              @click="moveTo(1)"
+            />
+          </div>
+        </div>
+      </AppIntroScreen>
+
+      <AppIntroScreen
+        v-if="step === 1"
+        :image="stepImage"
+        @back="moveTo(0)"
+        @next="moveTo(2)"
+        @skip="done"
+      >
+        <div
+          slot="content"
+          class="font-medium font-xl px-16 pt-16 m4-8 sm:px-10 sm:pt-10"
```

Comment:
Must be replicated to elements below. However, I think it should be `text-lg`.

```suggestion
          class="font-medium text-xl px-16 pt-16 m4-8 sm:px-10 sm:pt-10"
```